packages/block-directory/src/components/block-ratings/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * WordPress dependencies
+ */
+import { _n, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Stars from './stars';
+
+export const BlockRatings = ( { rating, ratingCount } ) => (
+	<div className="block-directory-block-ratings">
+		<Stars rating={ rating } />
+		<span
+			className="block-directory-block-ratings__rating-count"
+			aria-label={
```

Comment:
Is this `aria-label` necessary here? This item isn't focusable so I think this will never be announced. I see something similar applied for `div` later in the code. Am I missing something?

Reply:
I think the `aria-label` may still be necessary. The screen readers can still navigate to these elements even though they are not focusable.